lib/cc/yaml/nodes/root.rb

```diff
@@ -16,6 +20,17 @@ def nested_warnings(*)
         def inspect
           "#<#{self.class.name}:#{super}>"
         end
+
+        def exclude?(path)
+          exclude_paths.present? &&
+            exclude_paths.match_any?(path)
+        end
+
+        def rate?(path)
+          ratings.nil? ||
+            ratings.paths.blank? ||
```

Comment:
I hesitated here b/c the semantics between `#any?` and an empty array are actually important here and `#blank?` is less transparent than `#nil? || #empty?`. Can I get a second opinion?